provider-ci/internal/pkg/templates/bridged-provider/scripts/upstream.sh

```diff
@@ -172,8 +187,9 @@ end_rebase() {
 
   rm patches/*.patch
   cd upstream
-  git format-patch local -o ../patches
+  git format-patch local -o ../patches --no-prefix --zero-commit --no-signature --no-stat
   rm ../rebase-in-progress
+  git submodule update --force --init
```

Comment:
This should just call the function to reset upstream.
```suggestion
  apply "$1"
```

This has the nice property of being DRY and asserting that the new patch set applies cleanly.